build.ps1

```diff
@@ -52,6 +52,10 @@ try {
         Get-ChildItem -Path $PSScriptRoot\bench -Filter "BenchmarkDotNet.Artifacts" -Directory -Recurse | Move-Item -Destination "$PSScriptRoot\artifacts\benchmarks"
     }
 
+    # Generate Coverage Report
```

Comment:
Is this going to break folks on Windows trying to run the build? Should we somehow check to see if this is running in AppVeyor before invoking this? Probably shouldn't run from a local build if the script gets executed.

Reply:
Checking for the variable `$CI` now as described in the docs for the [built-in environment-variables](https://www.appveyor.com/docs/environment-variables/).